packages/cf-wallet.js/package.json

```diff
@@ -15,11 +15,11 @@
   "repository": "https://github.com/counterfactual/monorepo/tree/master/packages/cf.js",
   "license": "MIT",
   "scripts": {
-    "build": "tsc -b . && rollup -c",
+    "build": "tsc -b . && rollup -c && yarn docs",
```

Comment:
Maybe have a separate `docgen` script?

Reply:
We currently have a `docs` script which works independently of the other build commands. I'm just calling it from here to ensure that the docs get regenerated whenever we make a change to these packages. I could remove it, but I'm worried about us failing to regenerate the docs if we have to do so manually.